### ISSUE TITLE
Why is test acc the best of all epochs?

### ISSUE THREAD

**Issue by yipenghe** [vikasverma1077/GraphMix#2]
Hello, thanks for making the code public. 
The code here https://github.com/vikasverma1077/GraphMix/blob/master/semisupervised/codes/train.py#L238 reports the test acc based on the best dev acc of all epochs, shouldn't it be something like testing after the last epoch?

**Comment by vikasverma1077** [comment]
Standard practice is to use the dev ( validation set) for model selection and use the best model ( based on dev data) to report the test accuracy.
